apps/meteor/server/services/messages/service.ts

```diff
@@ -129,6 +129,36 @@ export class MessageService extends ServiceClassInternal implements IMessageServ
 		return result.insertedId;
 	}
 
+	async updateUserReferences({
+		username,
+		previousUsername,
+		userId,
+	}: {
+		username: string;
+		previousUsername: string;
+		userId: string;
+	}): Promise<void> {
```

Comment:
```suggestion
	async updateUserReferences(username: string, previousUsername: string, userId: string): Promise<void> {
```

Since the fields are all required, I think we can have normal params instead of an object here.